### PR TITLE
Update .yaml files for hpctookit 2020-03 installation on summit

### PR DESCRIPTION
**Pull request by mxz297**

### CODE REVIEW

File: ornl/summit/summit.config.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 config:
   # This is the path to the root of the Spack install tree.
   # You can use $spack here to refer to the root of the spack instance.
-  install_tree: /gpfs/alpine/csc322/world-shared/spack-install/packages
+  install_tree: /ccs/home/xmeng/hpctoolkit-install/packages
```
Comment (jmellorcrummey) [inline ornl/summit/summit.config.yaml:19]: I don't think that you want to put this in /ccs/home. This is an NFS file system. I think that you want the install directory to be in /gpfs. I think we have a project that you are part of where there is project space in /gpfs. Without being logged into summit, I am not sure where that is.

File: ornl/summit/summit.config.yaml [hunk 2]
```diff
@@ -29,10 +29,8 @@ config:
 
   # Locations where different types of modules should be installed.
   module_roots:
-    tcl:     /gpfs/alpine/csc322/world-shared/spack-install/module-roots/modules
-    lmod:   /gpfs/alpine/csc322/world-shared/spack-install/module-roots/lmod
-    dotkit: /gpfs/alpine/csc322/world-shared/spack-install/module-roots/dotkit
-
+    tcl:    /ccs/home/xmeng/hpctoolkit-install/module-roots/modules
```
Comment (jmellorcrummey) [inline ornl/summit/summit.config.yaml:32]: same comment as above

**Review by jmellorcrummey — CHANGES_REQUESTED** [review]

File: ornl/summit/summit.config.yaml (hunk 1 above)
Comment (mxz297) [reply ornl/summit/summit.config.yaml:19]: @jmellorcrummey I think you are referring to  `/gpfs/alpine/csc322/world-shared`, which has the following permission setting:

`drwxrwsr-x  3 johnmc csc322   4096 Apr 26  2019 hpctoolkit`

I cannot create new directory there, so it looks like that I am not in the `csc322` user group.

File: ornl/summit/summit.config.yaml (hunk 2 above)
Comment (jmellorcrummey) [reply ornl/summit/summit.config.yaml:32]: run groups on summit and let me know what groups you are in.

Comment (mxz297) [reply ornl/summit/summit.config.yaml:32]:
```
[xmeng@login4.summit ~]$ groups
xmeng gen010
```

**Review by jmellorcrummey — APPROVED** [review]

**Pull request merged**